packages/celotool/src/lib/utils.ts

```diff
@@ -300,3 +300,6 @@ export function addCeloGethMiddleware(argv: yargs.Argv) {
 export const validateAccountAddress = (address: string) => {
   return address !== null && address.toLowerCase().startsWith('0x') && address.length === 42 // 0x followed by 40 hex-chars
 }
+
+export const ensure0x = (hexstr: string) => (hexstr.startsWith('0x') ? hexstr : '0x' + hexstr)
+export const strip0x = (hexstr: string) => (hexstr.startsWith('0x') ? hexstr.slice(2) : hexstr)
```

Comment:
I've written a version of this, it's `stripHexLeader` in `utils/src/commentEncryption.ts`. Web3-utils also has a version of this. It'd be nice to use one set of functions for these conversions. I haven't seen another `ensure0x`  and I do like it.

Reply:
+1, we should at some point move these to `packages/utils` and use them everywhere

Reply:
I agree, i would rather keep this PR constrained to celotool for now. And along this with function, i believe there are many others we can extract.

i don't if commentEncryption is the right module for...maybe `util/hex` or `util/addresses`